lib/containers/common.pm

```diff
@@ -49,8 +49,8 @@ sub install_podman_when_needed {
             push(@pkgs, 'apparmor-parser')   if is_leap("=15.1");    # bsc#1123387
             zypper_call "in @pkgs";
         }
-        assert_script_run('podman info');
     }
+    assert_script_run('podman info');
```

Comment:
this is not relevant to the commit and also i wonder if there are corner cases where if is skipped but podman not installed. i would suggest to just leave it out of this PR

Reply:
what "corner cases" ? if `which podman` return 0 whatever will fail here should be a bug which we need to catch

Reply:
This change might be mentioned in the commit or in the description of this pull request but I'm also OK with keeping it here.

Reply:
i cant think any. but i think you are right. Can you at least make it a separate commit?